bigquery/google/cloud/bigquery/job.py

```diff
@@ -1374,9 +1374,9 @@ def input_file_bytes(self):
         :returns: the count (None until set from the server).
         :raises: ValueError for invalid value types.
         """
-        statistics = self._properties.get('statistics')
-        if statistics is not None:
-            return int(statistics['load']['inputFileBytes'])
+        return _helpers._get_sub_prop(
+                self._properties, ['statistics', 'load', 'inputFileBytes']
+            )
```

Comment:
Needs to wrap the return value in `_int_or_none`, because the back-end sends us strings.